### ISSUE TITLE
Can not save status product attribute through admin panel

### ISSUE THREAD

**Issue by adi3009** [magento/magento2#10823]
Spinning wheel of death when saving `status` product attribute through magento admin panel. Test magento instance has no modifications or extensions except sample data.

Looking into browser console, I see Internal server error response.

### Preconditions
Magento 2.1.7 CE and EE
Browser:- Chrome
OS: - Ubuntu 14.04

### Steps to reproduce
1. Login in admin panel.
2. Got to `Stores -> Attributes -> Product`
3. Edit `status` attribute
4. Click `Save Attribute` button

### Actual Result
![status - product attributes - attributes - stores - magento admin - google chrome_024](https://user-images.githubusercontent.com/3382618/30218625-668c3cba-94b1-11e7-8b06-c50933224486.png)

### Expected Result
no error in response

**Comment by orlangur** [comment]
Confirmed there is no proper handling of backend malfunction by client side, it can be emulated with
```
header('HTTP/1.1 503 Service Temporarily Unavailable');
        header('Status: 503 Service Temporarily Unavailable');
        header('Retry-After: 300');//300 seconds
        die;
```
added to `\Magento\Catalog\Controller\Adminhtml\Product\Attribute\Validate::execute` in 2.1.7.

However, `503 Service Temporarily Unavailable` status is a problem with your instance, not reproducible on my one with such steps and should be further investigated on your side.

**Comment by lbajsarowicz** [comment]
Tried to reproduce, but I failed. Could you provide more details, especially `var/report/###` file contents? If it's *503* error, it usually comes with response from server that contains mentioned `###` filename.

**Comment by orlangur** [comment]
@lbajsarowicz did you try to reproduce with the trick I provided? Basically, there are two issues here: 503 result which is most likely instance-specific (like some fatal error or uncleaned `var/di`) and the issue with incorrectt handling of 503 response on client side.

**Comment by magento-engcom-team** [comment]
@adi3009, thank you for your report.
The issue is already fixed in 2.1.8
